src/sql/workbench/contrib/queryplan2/browser/constants.ts

```diff
@@ -0,0 +1,254 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * Currently vscode automatically adds vscode-file://vscode-app/c:/src/azuredatastudio/out/vs/code/electron-browser/workbench/
+ * to all the icons fetched by the azdatagraph library. This is the relative path to the icons from that dir.
+ */
+let imageBasePath = '../../../../sql/workbench/contrib/queryplan2/browser/images/icons/'; //TODO: Make this an absolute path
```

Comment:
We should never have "absolute paths" in the code (how do you know where you're running?)

If you need to do something like for passing to the library or something you should be using FileAccess.asFileUri to convert to the file scheme if needed. Although since you're just passing in strings I'm not really sure what the problem is here.

Reply:
this won't work for package ADS.